CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Fixes
 
+- When Sentry is configured to capture Request bodies in ASP.NET Core, the uncompressed content is not captured when RequestDecompression middleware is enabled. Previously the request bodies were being captured prior to decompression ([#4315](https://github.com/getsentry/sentry-dotnet/pull/4315))
```

Comment:
thought: I was a bit confused when reading the CHANGELOG entry.

After my first read, I thought that no Content is captured anymore when `RequestDecompressionMiddleware` is present.
But actually the Content is now (tried to be) decompressed before capturing. Yet still capturing an Exception when decompression fails (without the Body, though).

I'm wondering if this could be phrased a bit differently 🤔.
I couldn't really come up with a clearer description that is not too verbose, though.